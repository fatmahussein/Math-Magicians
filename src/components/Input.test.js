import React from 'react';
import { render } from '@testing-library/react';
import Input from './input';

describe('Input Component', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Input data="123" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
