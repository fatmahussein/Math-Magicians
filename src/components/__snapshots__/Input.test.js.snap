// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component renders correctly 1`] = `
<DocumentFragment>
  <div
    class="input-wrapper"
    data-testid="initial"
  >
    123
  </div>
</DocumentFragment>
`;
